modules/serverless-installing-cli-linux-ibm-power.adoc

```diff
@@ -0,0 +1,93 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="installing-cli-linux_ibm_power_{context}"]
+= Installing the `kn` CLI for Linux on IBM Power Systems
```

Comment:
Tarball and RPM installs should be their own procedure modules, then included in an IBM power assembly. Please see RH modular docs templates for details.